perception/semantic_world/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ set(CMAKE_BUILD_TYPE Debug)
 set(MOVEIT_LIB_NAME moveit_semantic_world)
 
 add_library(${MOVEIT_LIB_NAME} src/semantic_world.cpp)
+add_dependencies(${MOVEIT_LIB_NAME} moveit_msgs_generate_messages_cpp )
```

Comment:
I think there was a better way to include this type of deps, but it escapes me right now. @dirk-thomas ?